pkgs/development/python-modules/flask-mysqldb/default.nix

```
@@ -0,0 +1,35 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, flask
+, mysqlclient
+}:
+
+buildPythonPackage rec {
+  pname = "flask-mysqldb";
+  version = "1.0.1";
+  format = "setuptools";
+
+  src = fetchFromGitHub {
+    owner = "alexferl";
+    repo = "flask-mysqldb";
+    rev = "v${version}";
+    hash = "sha256-qHPepvNA6oK2ZYAl9NqWepqwFSWOGu2KWuOJOe9V05g=";
+  };
+
+  propagatedBuildInputs = [
+    flask
+    mysqlclient
+  ];
+
+  pythonImportsCheck = [
+    "flask_mysqldb"
+  ];
+
+  meta = with lib; {
+    description = "MySQL connection support for Flask";
+    homepage = "https://github.com/alexferl/flask-mysqldb";
```

Tools (e.g., `nix-update`) can pick up the changelog entry during update and add the link to the changelog to the commit message.

```suggestion
    homepage = "https://github.com/alexferl/flask-mysqldb";
    changelog = "https://github.com/alexferl/flask-mysqldb/releases/tag/v${version}";
```

---

Fixed.